src/Enchantments.h

```diff
@@ -111,6 +111,9 @@ class cEnchantments
 
 	// tolua_end
 
+	//* Merge enchantments, check for valid enchantments on item type */
```

Comment:
The comment should begin with only one solidus (and one extra asterisk): `/** */`